pkg/git/ssh_config.go

```diff
@@ -67,10 +58,31 @@ func AddSSHConfig(cfg config.PipedGit) error {
 		}
 		cfgPath = path.Join(home, ".ssh", "config")
 	}
-	dir := filepath.Dir(cfgPath)
+	sshDir := filepath.Dir(cfgPath)
+
+	if err := os.MkdirAll(sshDir, 0700); err != nil {
+		return fmt.Errorf("failed to create a directory %s: %v", sshDir, err)
+	}
 
-	if err := os.MkdirAll(dir, 0700); err != nil {
-		return fmt.Errorf("failed to create a directory %s: %v", dir, err)
+	var sshKeyFile string
+	if cfg.SSHKeyFile != "" {
+		f, err := ioutil.TempFile(sshDir, "piped-ssh-key-*")
```

Comment:
The ioutil package itself is deprecated. Let's use `os.CreateTmep`: https://golang.org/pkg/os/#CreateTemp

Reply:
ioutil 🔫

Reply:
Nice catch! Applied this.